ostd/src/mm/page_table/boot_pt.rs

```diff
@@ -140,6 +140,7 @@ impl<E: PageTableEntryTrait, C: PagingConstsTrait> BootPageTable<E, C> {
             pte.set_prop(PageProperty::new(
                 prop.flags | PTE_POINTS_TO_FIRMWARE_PT,
                 prop.cache,
+                prop.priv_flags,
             ));
```

Comment:
I think it's better to just write:
```rust
let mut prop = pte.prop();
prop.flags |= PTE_POINTS_TO_FIRMWARE_PT;
pte.set_prop(prop);
```

So we don't really need the newly added `PageProperty::new` method.